tests/__init__.py

```diff
@@ -194,9 +194,8 @@ def test_getState(self):
                 universal_newlines=True,
             )
         with unittest.mock.patch("hddfancontrol.subprocess.check_output") as subprocess_check_output_mock:
-            subprocess_check_output_mock.side_effect = subprocess.CalledProcessError(0, "")
```

Comment:
This side effect should be kept, we are testing that a hdparm error leads to an `hddfancontrol.Drive.DriveState.UNKNOWN` state.

I suggest (untested):
```python
            subprocess_check_output_mock.side_effect = subprocess.CalledProcessError(0, "")
            self.assertEqual(self.drive.getState(), hddfancontrol.Drive.DriveState.UNKNOWN)
```

Reply:
Fixed. Please re-run the test because most of them failed with error:
```
coveralls.exception.CoverallsException: Could not submit coverage: 422 Client Error: Unprocessable Entity for url: https://coveralls.io/api/v1/jobs
```